cmd/query/app/server.go

```diff
@@ -349,11 +352,22 @@ func (s *Server) Close() error {
 	var errs []error
 	errs = append(errs, s.queryOptions.TLSGRPC.Close())
 	errs = append(errs, s.queryOptions.TLSHTTP.Close())
+
+	// Stop gRPC server
```

Comment:
this comment adds 0 new information
```suggestion
```

Reply:
Sorry , mb i will remove it to keep the code clean